pom.xml

```diff
@@ -356,6 +351,24 @@
                         <additionalJOption>-Xmaxwarns</additionalJOption>
                         <additionalJOption>65536</additionalJOption>
                     </additionalJOptions>
+                    <archive>
+                        <manifest>
+                            <addDefaultImplementationEntries>true</addDefaultImplementationEntries><!-- https://maven.apache.org/shared/maven-archiver/index.html#manifest -->
+                        </manifest>
+                        <manifestEntries>
+                            <Archive-Type>javadoc</Archive-Type>
+                            <Implementation-URL>${project.organization.url}</Implementation-URL>
+                            <Build-Java-Target>${maven.compiler.release}</Build-Java-Target>
+                            <SCM-Git-Commit-ID>${git.commit.id}</SCM-Git-Commit-ID>
+                            <SCM-Git-Commit-ID-Abbrev>${git.commit.id.abbrev}</SCM-Git-Commit-ID-Abbrev>
+                            <SCM-Git-Commit-Timestamp>${git.commit.time}</SCM-Git-Commit-Timestamp>
+                            <SCM-Git-Commit-Dirty>${git.dirty}</SCM-Git-Commit-Dirty>
+                            <SCM-Git-Remote-Origin-URL>${git.remote.origin.url}</SCM-Git-Remote-Origin-URL>
```

Comment:
This is also environment-specific. I saw one instance where it could vary. See https://github.com/chains-project/reproducible-central/issues/19#issuecomment-2581377157.
```suggestion
```

Reply:
on this one, I disagree: yes, people can configure in a strange way
but Reproducible Builds is not "get the same output whatever the builder did"
it's "get the same output with a reasonable effort on defining a near-enough environment"